RCNN_tf.py

```diff
@@ -474,6 +476,7 @@ def __init__(
 
     def train(self, train_data_feats, train_data_labels, val_data_feats,
               test_data_feats, test_data_labels, val_data_labels,
+              learning_depth=3, test_depth=6,
```

Comment:
Personally, I would find ```training_depth``` a more apt naming than ```learning_depth```.

Reply:
Can do, but it makes it easier for the auto completion. But if you feel happy with it